CodeStarterCamp_Week1/Pepero.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Pepero.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by 황지웅 on 2022/04/15.
+//
+import Foundation
+
+func makePeperoBar(peperoHeight: Int) {
+    let rangeOfPeperoHeight = 0..<peperoHeight
+
+    for _ in rangeOfPeperoHeight {
+        print(" | |")
+    }
+}
+
+func makePeperoBody(peperoBody: String, peperoToping: String, peperoBodyHeight: Int) {
+    let rangeOfPeperoHeight = 0..<peperoBodyHeight
+    for height in rangeOfPeperoHeight {
+        makePeperoBodyLayer(peperoBody: peperoBody, peperoToping: peperoToping, peperoHeight: height)
+    }
+}
+
+func makePeperoBodyLayer(peperoBody: String, peperoToping: String, peperoHeight: Int) {
+    if peperoHeight % 2 == 0 {
+        print("\(peperoToping + peperoBody)")
+    }else {
+        print(" \(peperoBody + peperoToping)")
+    }
+}
+
+
+
+func makePepero(peperoBodyLength: Int, peperoBodyContent: String, pepeproBodyTopingContent: String, peperoBarLength: Int) {
+    print("<정보>")
+    print("몸통: \(peperoBodyContent)")
+    print("토핑: \(pepeproBodyTopingContent)")
+    print("막대길이: \(peperoBarLength)")
+    
+    makePeperoBody(peperoBody: peperoBodyContent, peperoToping: pepeproBodyTopingContent, peperoBodyHeight: peperoBodyLength)
+    makePeperoBar(peperoHeight: peperoBarLength)
+    
+    print()
+}
+
+makePepero(peperoBodyLength: 10, peperoBodyContent: "***", pepeproBodyTopingContent: " ", peperoBarLength: 4)
```

Comment:
크~게 중요한 건 아니지만 Topping에 빈 공백 문자열을 넣어주셨는데 
Step3 요구사항에 나와있는 실행예시를 보시면 아래와 같이 써있어요!
> (일반 빼빼로의 좌측에는 빈칸이 하나 추가되어있습니다.)

```suggestion
makePepero(peperoBodyLength: 10, peperoBodyContent: " ***", pepeproBodyTopingContent: "", peperoBarLength: 4)
```
아마도 요것이지 아닐까 싶습니다ㅎㅎ

Reply:
만약 이부분이 파라미터로 " ***" 같이 처리 된다면 일관성있는 케이스로 보여지진않아서
함수 내부에서 처리 하는걸로 바꾸겠습니다.